src/core/schedule.js

```diff
@@ -25,17 +17,7 @@ export const getScheduleOutput = (lessonType, { start, end, title, description,
 	return `\n\n⏰ ${startTime} - ${endTime}\n${lessonType}:\n<b>${lessonTitle}</b>\n${lessonDescription}\n${audienceInfo}`
 }
 
-/**
- * Retrieves information about the lessons for the specified day.
- *
- * @param {string} targetDay - The target day for which lessons information is required.
- * @param {Array} scheduleData - The array of objects containing schedule data.
- * @param {boolean} shouldShiftWeek - Flag defining today is Sunday and the user has clicked the 'Schedule for tomorrow' button.
- * @param {number} weekNumber - The current week number.
- * @returns {string} - Formatted lessons information or NO_LESSONS_MESSAGE if no lessons found.
- */
-
-export function getLessonsForDay(targetDay, scheduleData, shouldShiftWeek = false, weekNumber = getWeekNumber(new Date())) {
+export function getLessonsForDay(targetDay, scheduleData, shouldShiftWeek, weekNumber = getWeekNumber(new Date())) {
 	const lessonToday = scheduleData.find(({ day, events }) => day === targetDay && events)
```

Comment:
I duplicate this handler in different parts of the code

Reply:
Hm, where?)

Reply:
Sorry, I moved the handler, but did not move the comment, thereby causing confusion. I mean `const statusOffset = JSON.parse(Object.values(weekOffset).join())`